assesspy/metrics.py

```diff
@@ -0,0 +1,375 @@
+import math
+from typing import Union
+
+import pandas as pd
+import statsmodels.api as sm
+
+from .utils import check_inputs
+
+
+def cod(
+    estimate: Union[list[int], list[float], pd.Series],
+    sale_price: Union[list[int], list[float], pd.Series],
+) -> float:
```

Comment:
`cod()` now gets 2 input args instead of the single `ratio` arg it took before. FINALLY.